packages/st2/debian/control

```diff
@@ -13,7 +13,7 @@ Vcs-Browser: https://github.com/stackstorm/st2
 Package: st2
 Architecture: any
 Pre-Depends: dpkg (>= 1.16.16), ${pre:Depends}, ${misc:Pre-Depends}, adduser
-Depends: ${shlibs:Depends}, ${misc:Depends}, sudo, adduser, ${Depends}, libssl-dev, libffi-dev, git, libpam0g, openssh-server, openssh-client, bash, netbase
+Depends: ${shlibs:Depends}, ${misc:Depends}, sudo, adduser, ${Depends}, libssl-dev, libffi-dev, git, libpam0g, openssh-server, openssh-client, bash, netbase, libldap2-dev, libsasl2-dev
```

Comment:
Per https://github.com/StackStorm/st2/pull/5081#issuecomment-729654906
we'll need to add these dependencies in `Build-Depends:` as they're required only for build.

Reply:
Updated

Reply:
:+1: